.cirrus.yml

```diff
@@ -2,6 +2,20 @@ container:
   image: gcr.io/flutter-cirrus/build-website-image:latest
 
 website_task:
+  # Only run this one on PR builds.
+  only_if: $CIRRUS_PR != ''
```

Comment:
AFAICT, there is no `CIRRUS_PR` environment variable in PR builds that originate from a branch on the website repo (see below for an example list of variables), and it would seem that `$CIRRUS_PR != ''` isn't true in that case.

```nocode
CIRRUS_BRANCH=chalin-jekyll-assets-0828
CIRRUS_BUILD_ID=6358354465128448
CIRRUS_CHANGE_IN_REPO=d1b309d7e8a23dd89c21bf3c3d5c32bbc7199c8a
CIRRUS_CI=true
CIRRUS_DEFAULT_BRANCH=master
CIRRUS_HTTP_CACHE_HOST=127.0.0.1:12321
CIRRUS_OS=linux
CIRRUS_REPO_CLONE_HOST=github.com
CIRRUS_REPO_CLONE_URL=https://github.com/flutter/website.git
CIRRUS_REPO_FULL_NAME=flutter/website
CIRRUS_REPO_NAME=website
CIRRUS_REPO_OWNER=flutter
CIRRUS_TASK_ID=6616932836966400
CIRRUS_TASK_NAME=website
CIRRUS_WORKING_DIR=/opt/website
```